### PR TITLE
OnboardingModal redesign

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

**Comment by JustCaptcha** [comment]
image
Desktop

**Comment by JustCaptcha** [comment]
image
Iphone 5/SE

File: universal-login-react/src/ui/Onboarding/OnboardingModal.tsx [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+import {ModalProgressBar} from '../commons/ModalProgressBar';
+import '../styles/base/onboardingModal.sass';
+import '../styles/themes/UniLogin/onboardingModalThemeUniLogin.sass';
 
 export function OnboardingModal(props: OnboardingProps) {
   return (
-    <div className="onboarding-modal-container">
-      <div className="onboarding-modal-content">
-        <p style={{color: 'white', fontSize: 20, textAlign: 'center'}}>
+    <div className={useClassFor('onboarding-modal')}>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Onboarding/OnboardingModal.tsx]: We can refactor it in the next PR:
```ts
const OnboardingModalWrapper = ({children}: OboardingProps) => 
    <div ...>
        <CompanyLogo/>
        ...
        {children}
        ...
    </div>


const OnboardingModal = () => 
    <OnboardingModalWrapper>
        <Onboarding
            sdk={sdk}
            walletService={walletService}
            domains={['mylogin.eth', 'universal-id.eth']}
        />
    </OnboardingModalWrapper>
```

File: universal-login-react/src/ui/Onboarding/Onboarding.tsx [hunk 2]
```diff
@@ -33,7 +34,7 @@ export const Onboarding = (props: OnboardingProps) => {
               exact
               path="/selector"
               render={({history}) =>
-                <>
+                <OnboardingModalWrapper hideModal={() => console.log('hide modal')} message={props.sdk.getNotice()}>
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/Onboarding.tsx]: remove console.log

File: universal-login-react/src/ui/commons/ModalProgressBar.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import {useClassFor, classForComponent} from '../utils/classFor';
+import '../styles/base/modalProgressBar.sass';
+import '../styles/themes/UniLogin/modalProgressBarThemeUniLogin.sass';
+
+interface ModalProgressBarProps {
+  progress?: number;
+  steps?: number;
+}
+
+export const ModalProgressBar = ({steps, progress}: ModalProgressBarProps) => {
```
Comment (xitronix) [inline universal-login-react/src/ui/commons/ModalProgressBar.tsx]: should we need it for now?

**Pull request merged**